evm/src/executor/fork/backend.rs

```diff
@@ -645,7 +646,7 @@ impl DatabaseRef for SharedBackend {
     fn basic(&self, address: B160) -> Result<Option<AccountInfo>, Self::Error> {
         trace!( target: "sharedbackend", "request basic {:?}", address);
         self.do_get_basic(b160_to_h160(address)).map_err(|err| {
-            error!(target: "sharedbackend",  ?err, ?address,  "Failed to send/recv `basic`");
+            DatabaseErrorLog::Basic(&address).log(&err);
```

Comment:
So this works nicely but it's an extra allocation—not too much of an optimizor but I preferred the simplicity of using the `error!` macro only—could we revert back to it and just introduce a check for if it's a non-archive node?

Reply:
resolved in e89915c6d57732244e5460a6762ac15a723d6843